test/unit/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -181,22 +181,6 @@ void _nrn_mechanism_register_data_fields(Args&&... args) {
             REQUIRE_THAT(generated,
                          ContainsSubstring(reindent_and_trim_text(expected_global_variables)));
         }
-        THEN("Correct range variables' macros are printed") {
-            std::string expected_range_macros = R"(/* NEURON RANGE variables macro definitions */
-    #define g(id) _ml->template fpfield<0>(id)
-    #define e(id) _ml->template fpfield<1>(id)
-    #define i(id) _ml->template fpfield<2>(id)
-    #define ar(id) _ml->template data_array<3, 2>(id)
-    #define s(id) _ml->template fpfield<4>(id)
-    #define ena(id) _ml->template fpfield<5>(id)
-    #define ina(id) _ml->template fpfield<6>(id)
-    #define Ds(id) _ml->template fpfield<7>(id)
-    #define v_unused(id) _ml->template fpfield<8>(id)
-    #define g_unused(id) _ml->template fpfield<9>(id))";
-
-            REQUIRE_THAT(generated,
-                         ContainsSubstring(reindent_and_trim_text(expected_range_macros)));
-        }
```

Comment:
"logically" macros should have been replaced with checking instance struct check? But with yesterday's discussion, do you plan to not put such checks here but somewhere else?

Reply:
I don't see how the mod file can compile if the instance is missing. Therefore, since I'm having difficulty pinning down what this test contributed, I simply removed it.

Reply:
Oh! I remember the reason for such type of test (from coreneuron codegen) - the order in which the variables were printed was introducing subtle bugs and it was pain to debug! i.e. nocmodl was printing `ena, ina` and new nmodl was printing `ina, ena`.

But with neuron and coreneuron codegen coming from new NMODL, we will have no/less issues. It's ok if you remove them from here. The ordering aspect could be tested in other ways.

Reply:
That does sounds like very limited fun.